ui-v2/src/components/ui/empty-state.tsx

```diff
@@ -0,0 +1,40 @@
+import { Card, CardContent } from "@/components/ui/card";
+import { icons } from "lucide-react";
+
+const EmptyStateIcon = ({ id }: { id: keyof typeof icons }): JSX.Element => {
+	const LucideIcon = icons[id];
+	return <LucideIcon className="h-12 w-12 text-muted-foreground mb-8" />;
+};
+const EmptyStateTitle = ({
+	children,
+}: { children: React.ReactNode }): JSX.Element => (
+	<h3 className="text-2xl font-bold">{children}</h3>
+);
+
+const EmptyStateDescription = ({
+	children,
+}: { children: React.ReactNode }): JSX.Element => (
+	<p className="text-md text-muted-foreground">{children}</p>
+);
+
+const EmptyStateActions = ({
+	children,
+}: { children: React.ReactNode }): JSX.Element => (
+	<div className="flex gap-2 mt-4">{children}</div>
+);
+
+type Props = {
+	children: React.ReactNode;
+};
+export const EmptyState = ({ children }: Props): JSX.Element => (
+	<Card>
+		<CardContent className="flex flex-col gap-2 items-center justify-center py-16">
+			{children}
+		</CardContent>
+	</Card>
+);
+
+EmptyState.Icon = EmptyStateIcon;
+EmptyState.Title = EmptyStateTitle;
+EmptyState.Description = EmptyStateDescription;
+EmptyState.Actions = EmptyStateActions;
```

Comment:
This is similar to the pattern the `shadcn/ui` components we're using, but those modules will export each component individually (like in `ui-v2/src/components/ui/card.tsx`). I think we should be consistent with how we expose sets of collaborative components. We could update our other components to follow this pattern, but I'd prefer to follow the `shadcn/ui` style, so we don't diverge too much from the `shadcn/ui` source. Let me know if you feel strongly the other way and we can discuss!

Reply:
sounds good. Let me make an update

Reply:
Updated in latest push